forage-android/src/main/java/com/joinforage/forage/android/ui/ForagePINEditText.kt

```diff
@@ -46,6 +46,10 @@ class ForagePINEditText @JvmOverloads constructor(
         logger.i("ForagePINEditText successfully rendered")
     }
 
+    override fun clearText() {
+        vault.clearText()
```

Comment:
BT is the only SDK that has an opinionated way of doing this (see [docs](https://developers.basistheory.com/docs/sdks/mobile/android/types#:~:text=setText(value%3A%20String%3F)). That's why its not `.setText("")` like it is for VGS and PAN